### PR TITLE
Handle RequestScopedTransaction transactions

### PR DESCRIPTION
**Pull request by wcekan**
Process mutiplexed RequestScopedTransactions

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by clayreimann** [comment]
LGTM 👍

File: elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java [hunk 1]
```diff
@@ -64,6 +67,7 @@
     private final boolean isScrollEnabled;
     private final ScrollMode scrollMode;
     @Getter(value = AccessLevel.PROTECTED)
+    @Setter
```
Comment (clayreimann) [inline elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java]: Does this also want `AccessLevel` (or is it inherited from the getter)?

**Comment by aklish** [comment]
Let's add a test for it though.

File: elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java (hunk 1 above)
Comment (wcekan) [inline elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java]: Setter must be public, which is the default AccessLevel.

**Comment by wcekan** [comment]
test added.  If you reverse the change the tests fails with
```
Failed tests: 
  ResourceIT.createChildRelateExisting:962 1 expectation failed.
Expected status code <200> doesn't match actual status code <400>.
```

**Comment by aklish** [comment]
👍

**Pull request merged**